### PR TITLE
DT-916 Added ability install from pre-compiled binary

### PR DESCRIPTION
**Pull request by hartfordfive**

### CODE REVIEW

File: recipes/source.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  directory node['stats_ag']['metrics_dir'] do
+    owner 'root'
+    group 'root'
+    mode '0755'
+    action :create
+    recursive true
+  end
+
+  metrics_list = ['cpu','disk','host','load','memory','net']
```
Comment (ls-yann-david) [inline recipes/source.rb]: This is repeated in both recipes, it should be a third one, that's always included.

File: recipes/source.rb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  end
+
+  cron 'run stats-ag every minute' do
+    action :delete if node['stats_ag']['enable'] == false 
+    hour '*'
+    minute '*'
+    command "#{node['stats_ag']['base_dir']}/stats-ag -m #{node['stats_ag']['metrics_dir']} -s #{node['stats_ag']['scripts_dir']} -p #{node['stats_ag']['date_prefix_format']} > #{node['stats_ag']['log_file']} 2>&1"
+  end
+
+  metrics_list.each do |type|
```
Comment (ls-yann-david) [inline recipes/source.rb]: The cron and the log rotate also.

Comment (hartfordfive) [inline recipes/source.rb]: Yeah, I did this quick although I just realized that code duplication.  I'll convert those into other recipes.

**Comment by thanhliemtran** [comment]
👍

**Pull request merged**